connect/resources/template.py

```diff
@@ -15,9 +14,22 @@ class TemplateResource(BaseResource):
 
     resource = 'templates'
 
-    def list(self, filters=None):
+    def list(self, product_id):
+        """ List an activation template.
+        :param str product_id: Primary key of the product to search for.
+        :return: response object with templates contents.
+        """
+        """
         # type: (Dict[str, Any]) -> List[Any]
-        raise AttributeError('This resource do not have method `list`')
+        """
+        if not product_id:
+            raise ValueError('Invalid product Id for list template')
+        response, _ = self._api.get(
+            '/public/v1/products/'+product_id+'/templates/',
```

Comment:
Please add spaces between variables/literals and the "+" operator.

Ex:
'/public/v1/products/' + product_id + '/templates/',

Reply:
Done, thanks!